quadra/models/evaluation.py

```diff
@@ -66,7 +66,7 @@ class TorchscriptEvaluationModel(BaseEvaluationModel):
     """Wrapper for torchscript models."""
 
     def __call__(self, *args: Any, **kwargs: Any) -> Any:
-        return self.model(*args, **kwargs)
+        return self.model(*args, **kwargs, batch_imagenet=None)
```

Comment:
Smartest way to manage this? Obviously this additional parameter is needed only for logical_anomaly model's forward. Also, what we do here should be done in other types' _call_ functions